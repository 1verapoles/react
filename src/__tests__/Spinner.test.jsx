import React from 'react';
import { render } from '@testing-library/react';
import Spinner from '../Spinner';

describe('Testing Spinner component', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<Spinner />);

    expect(asFragment()).toMatchSnapshot();
  });
});
