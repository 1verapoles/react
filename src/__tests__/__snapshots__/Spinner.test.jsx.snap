// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Spinner component renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="d-flex justify-content-center"
  >
    <div
      class="spinner-border text-primary"
      role="status"
    >
      <span
        class="sr-only"
      >
        Loading...
      </span>
    </div>
  </div>
</DocumentFragment>
`;
